conformant = "Unsupported"
notes = """
Does not support TypeVarTuple.
"""
output = """
File "generics_typevartuple_concat.py", line 9, in <module>: typing.TypeVarTuple not supported yet [not-supported-yet]
File "generics_typevartuple_concat.py", line 17, in <module>: Function TypeVarTuple.__init__ expects 1 arg(s), got 2 [wrong-arg-count]
File "generics_typevartuple_concat.py", line 18, in <module>: Function TypeVarTuple.__init__ expects 1 arg(s), got 2 [wrong-arg-count]
File "generics_typevartuple_concat.py", line 22, in <module>: Invalid type annotation 'Generic'  [invalid-annotation]
File "generics_typevartuple_concat.py", line 26, in <module>: class Array is not indexable [not-indexable]
File "generics_typevartuple_concat.py", line 27, in add_batch_axis: bad return type [bad-return-type]
File "generics_typevartuple_concat.py", line 30, in <module>: class Array is not indexable [not-indexable]
File "generics_typevartuple_concat.py", line 31, in del_batch_axis: bad return type [bad-return-type]
File "generics_typevartuple_concat.py", line 34, in <module>: class Array is not indexable [not-indexable]
File "generics_typevartuple_concat.py", line 35, in add_batch_channels: bad return type [bad-return-type]
File "generics_typevartuple_concat.py", line 38, in <module>: class Array is not indexable [not-indexable]
File "generics_typevartuple_concat.py", line 40, in func1: class Array is not indexable [not-indexable]
File "generics_typevartuple_concat.py", line 42, in func1: class Array is not indexable [not-indexable]
File "generics_typevartuple_concat.py", line 44, in func1: class Array is not indexable [not-indexable]
File "generics_typevartuple_concat.py", line 47, in <module>: Invalid type annotation 'T'  [invalid-annotation]
File "generics_typevartuple_concat.py", line 47, in <module>: Invalid type annotation '<instance of Tuple[nothing, ...]>'  [invalid-annotation]
File "generics_typevartuple_concat.py", line 47, in <module>: Invalid type annotation '<instance of Tuple[TypeVar, ...]>'  [invalid-annotation]
File "generics_typevartuple_concat.py", line 48, in prefix_tuple: bad return type [bad-return-type]
File "generics_typevartuple_concat.py", line 51, in <module>: Function prefix_tuple was called with the wrong arguments [wrong-arg-types]
File "generics_typevartuple_concat.py", line 52, in <module>: Any [assert-type]
File "generics_typevartuple_concat.py", line 55, in <module>: Invalid type annotation '<instance of Tuple[nothing, ...]>'  [invalid-annotation]
File "generics_typevartuple_concat.py", line 55, in <module>: Invalid type annotation '<instance of Tuple[TypeVar, ...]>'  [invalid-annotation]
"""
